api/browsers.go

```diff
@@ -1,3 +1,6 @@
+package api
+
+const browsersJSON = `
```

Comment:
This change is somewhat important, but it's not in the PR description. Could you make sure to explain it in both the PR description and the final merge commit?